### DISCUSSION TITLE
Thank you so much

### DISCUSSION THREAD

**Discussion by mx** [spakin/SimpInkScr#109] [General]
Just wanted to say what an absolute lifesaver this extension was for me. I'm generating some UV-printing artwork that needs a bunch of stuff aligned with holes, and the alignment guide given to me was with the (0, 0) coordinate in the center. Trivial in a CAD program, but Inkscape makes it difficult since (0, 0) is always in the upper-left point (even after a view rotation, it just rotates that center point). With your extension I was able to define all my hole center points and radiuses and do the coordinate transformation in Python, and then generate the geometry there. I'm sure there's some other way to do it quickly in Inkscape but as an Inkscape newbie and CAD veteran, this got the job done for me really quickly in comparison. Thanks so much!

**Comment by spakin** [comment]
I'm glad to hear you find Simple Inkscape Scripting useful!  Thanks for reporting your experience.
